src/Mvc/test/ViewCompilationApps/Directory.Build.props

```diff
@@ -0,0 +1,18 @@
+<Project>
+  <!-- Skip the parent folder to prevent getting test package references. -->
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory)..\..\, Directory.Build.props))\Directory.Build.props" />
+
+  <PropertyGroup>
+    <!-- Required to build test asset projects -->
+    <IsTestAssetProject>true</IsTestAssetProject>
```

Comment:
Can you put these projects in a folder named 'testassets'? This property will be set automatically when the folder uses that name.

Reply:
I think with an updated layout I can do this. I was concerted with potentially collision with the testassets under the root Mvc/test/WebSites so I opted for a different directory name.